From c3910bb945aba02e0c06dec041da9f1f148b0df1 Mon Sep 17 00:00:00 2001
From: David Malcolm <dmalcolm@redhat.com>
Date: Thu, 2 Sep 2021 11:40:42 -0400
Subject: [PATCH 12/17] dts.exp: use /usr/bin/gcc

---
 libstdc++-v3/testsuite/lib/dts.exp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libstdc++-v3/testsuite/lib/dts.exp b/libstdc++-v3/testsuite/lib/dts.exp
index 76ece66d3..dc5cf3e95 100644
--- a/libstdc++-v3/testsuite/lib/dts.exp
+++ b/libstdc++-v3/testsuite/lib/dts.exp
@@ -11,9 +11,9 @@
 
 proc get_dts_base_version { } {
 
-    # Invoke gcc in the PATH to get at the underlying GCC version
+    # Invoke /usr/bin/gcc to get at the underlying GCC version
     # in dotted form (e.g. "4.8.5").
-    set dotted_version [exec gcc -dumpversion]
+    set dotted_version [exec /usr/bin/gcc -dumpversion]
     verbose "dotted_version: '$dotted_version'" 2
 
     # Extract major, minor, patchlevel
-- 
2.31.1

